packages/app/src/components/TimetableCourseCard.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div>
+    <div class="course-info-card">
+      <div class="course-card-header" :style="{ background: color }">
+        <h3 class="course-card-title">{{codeAndName}}</h3>
+        <div class="details-button">
+          <v-btn text>Details</v-btn>
+        </div>
+      </div>
+      <div class="course-info-header">
+        <v-row>
+          <v-col>
+            <h4>Activity</h4>
+          </v-col>
+          <v-col>
+            <h4>Time</h4>
+          </v-col>
+          <v-col>
+            <h4>Location</h4>
+          </v-col>
+          <v-col>
+            <h4>Instructor</h4>
+          </v-col>
+        </v-row>
+      </div>
+      <div class="sections-info">
+        <v-row v-for="meetingsection in meetingSections" :key="meetingsection.section">
+          <v-col cols="3">{{meetingsection.section}}</v-col>
+          <v-col>{{meetingsection.day}}</v-col>
+          <v-col>{{getFormattedTime(meetingsection.start, meetingsection.end)}}</v-col>
+          <v-col cols="3">{{meetingsection.location}}</v-col>
+          <v-col cols="3">{{meetingsection.instructorName}}</v-col>
+        </v-row>
+      </div>
+      <v-btn icon class="edit-button">
+        <v-icon>mdi-pencil-box-outline</v-icon>
+      </v-btn>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "TimetableCourseCard",
+  props: {
+    codeAndName: {
+        type: String,
+        default: () => "Undefined"
+    },
+    color: {
```

Comment:
I think the color should be dynamically assigned by the component as it is in the TimetableEvent component.

Reply:
The color can't be dynamically assigned in this component because the color has to match the color in TimetableEvent. The timetable component computes and assigns a color to each course, and that color needs to be passed into this component